### ISSUE TITLE
Incorrect HTTPS Check

### ISSUE THREAD

**Issue by PhilDakin** [ChuckJonas/next-ssl-redirect-middleware#6]
Chrome sends me a `http,https` `x-forwarded-proto` header so I feel the appropriate check is actually:

```
    const xfp = req.headers.get("x-forwarded-proto"); 
    const isHttps = xfp?.indexOf("https") !== -1;
```

Instead of the `!==` check currently done in the code.

**Comment by ChuckJonas** [comment]
thanks! Will make this update

**Comment by ChuckJonas** [comment]
Should be fixed in `0.1.5`
